test/test_irods_http_api.py

```diff
@@ -3339,6 +3339,90 @@ def test_report_operation(self):
         self.assertIn('zones', zone_report)
         self.assertGreaterEqual(len(zone_report['zones']), 1)
 
+    def test_adding_removing_and_modifying_zones(self):
+        headers = {'Authorization': 'Bearer ' + self.rodsadmin_bearer_token}
```

Comment:
Is there any value in showing that non-admin users cannot perform any of these action (as is being done in the test for adding, modifying, and removing resources)? "No" is an okay answer here since I'm guessing (hoping) the administration library or the admin API is enforcing this.

Reply:
Yes, the test should do that regardless of the internals just so we don't go backwards. We know non-admin users cannot do it today due to the current iRODS behavior, but that may not be the case a year (or more) from now.

Reply:
Done